src/pages/blog/2024-10-14-welcome-yaacov.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Welcoming Yaacov Rydzinski to the GraphQL.js Maintainer Team"
+tags: ["blog"]
+date: 2024-10-14
+byline: Jory Burson and Benjie Gillam
+---
+
+We are thrilled to announce a new co-maintainer of GraphQL.js: Yaacov Rydzinski
+[@Yaacov Rydzinski](https://github.com/yaacovCR) has been approved as a
+co-maintainer of GraphQL.js!
+
+Yaacov’s journey into open source and GraphQL is unique, having earned degrees
+in both Medicine and Computer Science. Whereas most maintainers are employed as
+software developers, Yaacov is a technical radiologist by day and star GraphQL
```

Comment:
```suggestion
software developers, Yaacov is a radiologist by day and star GraphQL
```

I think you probably meant teleradiologist, but plain ol' radiologist is probably better in this context.

Reply:
Otherwise, looks great!